gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -99,7 +99,7 @@
         </module>
         <module name="HideUtilityClassConstructor"/> <!-- Java Coding Guidelines: Private constructors -->
         <module name="IllegalImport"> <!-- Java Coding Guidelines: Import the canonical package -->
-            <property name="illegalPkgs" value="sun, org.elasticsearch.common.base, org.elasticsearch.common.joda"/>
+            <property name="illegalPkgs" value="sun, org.elasticsearch.common.base, org.elasticsearch.common.joda, jersey.repackaged.com.google.common, com.google.api.client.repackaged"/>
```

Comment:
looks like I missed this party, but sorting the list entries might be a good idea